selenium-test/src/main/java/com/bakdata/conquery/test/util/DragDropHelper.java

```diff
@@ -0,0 +1,302 @@
+package com.bakdata.conquery.test.util;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ *
+ * @author Marcus Baitz
+ */
+public class DragDropHelper {
+
+    public enum Position {
+        Top_Left,
+        Top,
+        Top_Right,
+        Left,
+        Center,
+        Right,
+        Bottom_Left,
+        Bottom,
+        Bottom_Right
+    }
+
+    public static int getX(Position pos, int width) {
+        if (Position.Top_Left.equals(pos) || Position.Left.equals(pos) || Position.Bottom_Left.equals(pos)) {
+            return 1;
+        } else if (Position.Top.equals(pos) || Position.Center.equals(pos) || Position.Bottom.equals(pos)) {
+            return width / 2;
+        } else if (Position.Top_Right.equals(pos) || Position.Right.equals(pos) || Position.Bottom_Right.equals(pos)) {
+            return width - 1;
+        } else {
+            return 0;
+        }
+    }
+
+    static int getY(Position pos, int height) {
+        if (Position.Top_Left.equals(pos) || Position.Top.equals(pos) || Position.Top_Right.equals(pos)) {
+            return 1;
+        } else if (Position.Left.equals(pos) || Position.Center.equals(pos) || Position.Right.equals(pos)) {
+            return height / 2;
+        } else if (Position.Bottom_Left.equals(pos) || Position.Bottom.equals(pos) || Position.Bottom_Right.equals(pos)) {
+            return height - 1;
+        } else {
+            return 0;
+        }
+    }
+
+    private static String javaScriptEventSimulator = ""
+            + /* Creates a drag event */ "function createDragEvent(eventName, options)\r\n"
+            + "{\r\n"
+            + "var event = document.createEvent('HTMLEvents');\r\n"
+            + "event.initEvent('DragEvent', true, true);\r\n"
+            + //" var event = document.createEvent(\"DragEvent\");\r\n" +
```

Comment:
Can this line be removed?

This is quite a lot of JavaScript code -- could this be loaded from a file to improve readability a bit?